README.md

```diff
@@ -0,0 +1,7 @@
+
+# Tensorboard
+By running `tensorboard --logdir=runs` a local instance of TensorBoard will be started (http://localhost:6007/)
```

Comment:
```suggestion
By running `tensorboard --logdir=runs` a local instance of TensorBoard will be started (http://localhost:6006/)
```
Minor remark: for most users it will be port `6006` and `6007` if the other one is in use